src/App.js

```diff
@@ -1,25 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          My App
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <div className="app">
+    {/* <h1 className="center">My Calculator</h1> */}
```

Comment:
- [ ] It is against the best practices to leave unused code as a comment in your codebase, I'll suggest you remove this as it makes your codebase a bit unprofessional. :smile: :wave: